src/sentry/web/urls.py

```diff
@@ -1012,7 +1012,7 @@
                     name="sentry-organization-crons",
                 ),
                 re_path(
-                    r"^(?P<organization_slug>[\w_-]+)/crons/(?P<project_slug>[\w_-]+)/(?P<monitor_slug>[\w_-]+)/$",
+                    r"^(?P<organization_slug>[\w_-]+)/crons/(?P<project_slug>[\w_-]+)/(?P<monitor_id_or_slug>[\w_-]+)/$",
```

Comment:
Why are we replacing web urls? Should it just be in api urls?

Reply:
yep you are right, this shouldn't be here